pkg/yurthub/cachemanager/cache_manager.go

```diff
@@ -248,20 +221,21 @@ func (cm *cacheManager) saveWatchObject(ctx context.Context, info *apirequest.Re
 
 	comp, _ := util.ClientComponentFrom(ctx)
 	reqContentType, _ := util.ReqContentTypeFrom(ctx)
-	serializers, err := cm.serializerManager.CreateSerializers(reqContentType, info.APIGroup, info.APIVersion)
+	serializers, err := cm.serializerManager.CreateSerializers(reqContentType, "", "v1", "events")
```

Comment:
I think we need a serializer for `WatchEvent`, so maybe need to use `cm.serializerManager.CreateSerializers(reqContentType, info.APIGroup, info.APIVersion, "watchevent")` to create serializer.

and we need to build a yurthub image and make sure the new image can work no problem before we make pr.

Reply:
Code is already updated, thanks for review